### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for  "Push Image dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1407]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. Observe the name property for the dialog

**Actual Result:**
Name property is empty for the Dialog.

**Narrator Behaviour:**
editing….

**NVDA:**
Push Image
Configure the container registry you want to push to.
edit
blank

**Expected Result:**
Name property should be "Push Image" dialog

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-dialog-inspect](https://user-images.githubusercontent.com/22956130/35685485-e28d0186-078f-11e8-89a5-7f5098036bba.jpg)
![push image-dialog-narrator](https://user-images.githubusercontent.com/22956130/35685488-e58f62ca-078f-11e8-8c9f-50c80076db89.jpg)
![push image-dialog-nvda](https://user-images.githubusercontent.com/22956130/35685491-e7a70e0a-078f-11e8-8d5a-4523c93f4443.jpg)

**Comment by jdneo** [comment]
Take Eclipse's official TitleAreaDialog as an example:

![image](https://user-images.githubusercontent.com/6193897/38715085-c95a855e-3f0b-11e8-9db8-95bde333ff99.png)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

See #1410 for more the related screenshot.

**Comment by jdneo** [comment]
Can show the `name` property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150067-ff85f100-d307-11e8-968d-e880a5ce3b05.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749880
